### ISSUE TITLE
(microsoft-windows-terminal) Support system wide installation

### ISSUE THREAD

**Issue by pascalberger** [mkevenaar/chocolatey-packages#77]
## Expected Behavior

Windows Terminal should be installed system wide to work seamlessly with Chocolatey and also in scenarios where background service is used.

## Current Behavior

Package uses `Add-AppxPackage` CmdLet which installs the app in user context, which means that the package doesn't work when background service is configured.

## Steps to Reproduce (for bugs)

1. Enable Chocolatey background service running under a different user than the user which will install the package
2. Install Terminal
3. Terminal won't be available for the user which initiated the installation
4.

## Your Environment

* Package Version used: 1.3.2651.0
* Operating System and version: Microsoft Windows [Version 10.0.19041.572]
* Chocolatey version: 0.10.15
* Install/uninstall gist:

**Comment by mkevenaar** [comment]
Hi @pascalberger 
Thanks for submitting this issue. I think it's a nice improvement that I think would be best as a package parameter.

Are you willing to submit a PR for this?

Thanks in advance,

Maurice

**Comment by pascalberger** [comment]
If I find a way to do this I for sure can submit a PR (let me know if you have any ideas).

Can implement it as a package parameter, but I would suggest to make system wide installation the default in this case, since Chocolatey is a system wide package manager (otherwise you would have the issue that User A installs the package, User B has the choco package installed, but doesn't have the app available)

**Comment by pascalberger** [comment]
System wide installation basically seems to work using the [Add-AppxProvisionedPackage](https://docs.microsoft.com/en-us/powershell/module/dism/add-appxprovisionedpackage?view=win10-ps) command. I'll try to create a PR based on this.

**Comment by pascalberger** [comment]
I tried with Add-AppxProvisionedPackage, but unfortunately couldn't get it relieable to work. It works one time, but then throws an error the other. Also documentation mentions that this will only install for new created accounts, but in my tests it was also added to existing accounts. Another thing is uninstallation, where documentation mentiones that this is also for new accounts. Not sure if there's a reliable way to do this 😞
